### ISSUE TITLE
[Bug]: Docker container not starting

### ISSUE THREAD

**Issue by mrtnbr** [bropat/eufy-security-ws#81]
### Version

0.8.0 and 0.8.1

### Node version

v16.13.1

### Operating System type

Docker

### Operating system version

Debian 10 release 10

### Describe the bug

With 0.8.0 and 0.8.1, the container won't start and gives the following error:

/usr/local/bin/docker-entrypoint.sh: exec: line 11: /usr/src/app/run.sh: not found

0.7.0 works without problems

### To reproduce

Start a Docker container with an image 0.8.0 or above.

**Comment by lolouk44** [comment]
Also present in 0.8.2

**Comment by mrtnbr** [comment]
I can confirm 0.8.2 does not fix this issue

**Comment by bropat** [comment]
@lolouk44 @mrtnbr @Aspyryan 

The bug was already fixed in version 0.8.0. Had deleted the image from dockerhub and recreated it. The newer images have never been changed since the build.

I think your mistake is that you missed to make a docker pull of the new image.

**Comment by Aspyryan** [comment]
The problem for me was caused when I pulled the project to my local pc on Windows and then tried to rebuild the docker image myself. My commit should fix that though. Correct me if I'm wrong

**Comment by lolouk44** [comment]
I have experienced the same CRLF/LF issue with some containers I built in the post. Not tested yet but totally makes sense to me.

**Comment by mrtnbr** [comment]
> @lolouk44 @mrtnbr @Aspyryan
> 
> The bug was already fixed in version 0.8.0. Had deleted the image from dockerhub and recreated it. The newer images have never been changed since the build.
> 
> I think your mistake is that you missed to make a docker pull of the new image.

I would be surprised if that is what happened, but I can't check it to make sure. I pulled the latest release just now and it works. Thanks!

**Comment by lolouk44** [comment]
so just pulled the latest version and logs are stating connection is not successful (works fine in 0.7.0)

<details>
  <summary>0.8.3 Logs</summary>

```  
2022-02-09 11:20:30.516  INFO  Eufy Security server listening on host 0.0.0.0, port 3000 
2022-02-09 11:20:31.583  ERROR Status return code 401, invalidate token 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:31.588  ERROR Status return code not 200 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:31.905  ERROR Status return code 401, invalidate token 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:31.906  ERROR Status return code not 200 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:32.234  ERROR Status return code 401, invalidate token 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:32.235  ERROR Status return code not 200 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:32.561  ERROR Status return code 401, invalidate token 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:32.562  ERROR Status return code not 200 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:32.923  ERROR Status return code 401, invalidate token 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:32.924  ERROR Status return code not 200 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:32.925  INFO  No stations found. 
2022-02-09 11:20:33.237  ERROR Status return code 401, invalidate token 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:33.238  ERROR Status return code not 200 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:20:33.239  INFO  No devices found. 
```
</details>

For comparison:
<details>
  <summary>0.7.0 Logs</summary>

```
2022-02-09 11:19:34.668  INFO  Eufy Security server listening on port 3000 
2022-02-09 11:19:35.563  ERROR Status return code 401, invalidate token 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:19:35.569  ERROR Status return code not 200 
{
  status: 401,
  statusText: 'Unauthorized'
}
2022-02-09 11:19:36.021  INFO  Switching to another API_BASE (https://security-app-eu.eufylife.com) and get new token. 
2022-02-09 11:19:38.512  INFO  Connected to station T8200N02204208EC on host 192.168.9.38 and port 27245 
2022-02-09 11:19:38.743  INFO  Push notification connection successfully established 
2022-02-09 11:19:46.418  INFO  Push notification connection successfully established   
```
</details>

Am I doing something wrong / missing something? Also @bropat do you need this moved to a new issue?
I have a wired Doorbell 2K for reference
